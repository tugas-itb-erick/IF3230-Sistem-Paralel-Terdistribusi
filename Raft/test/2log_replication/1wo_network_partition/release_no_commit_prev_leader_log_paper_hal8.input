5

set_timeout 1 0
simulate 10
client_send_request 1 + 1
simulate 10
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5

set_timeout 2 0
simulate 10
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5

set_timeout 1 1000
set_timeout 2 0
set_timeout 3 1000
set_timeout 4 1000
set_timeout 5 1000
disconnect 2 3
disconnect 2 4
disconnect 2 5
client_send_request 2 + 1
simulate 10
get_log 1
get_log 2
get_log 3
get_log 4
get_log 5
get_commit_index 1
get_commit_index 2
get_commit_index 3
get_commit_index 4
get_commit_index 5

connect 2 3
connect 2 4
connect 2 5
set_timeout 1 0
simulate 1000
get_log 1
get_log 2
get_log 3
get_log 4
get_log 5
get_commit_index 1
get_commit_index 2
get_commit_index 3
get_commit_index 4
get_commit_index 5
end_simulation
