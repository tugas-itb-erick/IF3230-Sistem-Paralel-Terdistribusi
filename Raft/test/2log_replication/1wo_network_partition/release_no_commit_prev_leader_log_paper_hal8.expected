S1 leader term:2 voted_for:1 commit_index:1 data:1 logs:T0[0] T2[+1]
S2 follower term:2 voted_for:1 commit_index:1 data:1 logs:T0[0] T2[+1]
S3 follower term:2 voted_for:1 commit_index:1 data:1 logs:T0[0] T2[+1]
S4 follower term:2 voted_for:1 commit_index:1 data:1 logs:T0[0] T2[+1]
S5 follower term:2 voted_for:1 commit_index:1 data:1 logs:T0[0] T2[+1]
S1 follower term:3 voted_for:2 commit_index:1 data:1 logs:T0[0] T2[+1]
S2 leader term:3 voted_for:2 commit_index:1 data:1 logs:T0[0] T2[+1]
S3 follower term:3 voted_for:2 commit_index:1 data:1 logs:T0[0] T2[+1]
S4 follower term:3 voted_for:2 commit_index:1 data:1 logs:T0[0] T2[+1]
S5 follower term:3 voted_for:2 commit_index:1 data:1 logs:T0[0] T2[+1]
T0[0] T2[+1] T3[+1]
T0[0] T2[+1] T3[+1]
T0[0] T2[+1]
T0[0] T2[+1]
T0[0] T2[+1]
1
1
1
1
1
T0[0] T2[+1] T3[+1]
T0[0] T2[+1] T3[+1]
T0[0] T2[+1] T3[+1]
T0[0] T2[+1] T3[+1]
T0[0] T2[+1] T3[+1]
1
1
1
1
1
